### PR TITLE
bugfix(issue 20043): Parameter save_relative_paths of tf.train.Saver do not work non-local filesystem.

### PR DESCRIPTION
**Pull request by jony0917**
Fix the bug on issue 20043:  parameter save_relative_paths of tf.train.Saver do not work non-local filesystem.

### CODE REVIEW

**Comment by jony0917** [comment]
I signed it!

**Comment by jony0917** [comment]
i am sure that i have signed a CLA with my gihub account name  "jony0917" as "GitHub Account Name" in the CLA form,  and i am sure that the email address  from "git config --global user.email" has been added to my github account, so do i miss something?

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 44 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 59 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 74 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 89 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 104 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 119 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 137 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 152 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 44 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 59 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 74 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 89 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 119 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 134 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 149 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 164 days with no activity and the `awaiting review` label has been applied.

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
